### ISSUE TITLE
Classes missing in documentation

### ISSUE THREAD

**Issue by moi90** [readthedocs/readthedocs.org#7703]
## Details

* Read the Docs project URL: https://morphocut.readthedocs.io/en/latest/image.html
* Build URL (if applicable): https://readthedocs.org/api/v2/build/12422667.txt
* Read the Docs username (if applicable): moi90
* Project respository: https://github.com/morphocut/morphocut/

## Expected Result

The linked page is about a bunch of classes in `morphocut.image`. All of them should show up in the documentation, like they do when building locally and on the `stable` version.

The problem is not only with this page, but also most of the others. It seems that the classes annotated with `@ReturnOutputs` and `@Output` are the problem but the implementation of these decorators did not change between `stable` and `master`.

## Actual Result

Only `RegionProperties` is shown.

**Comment by stsewd** [comment]
I think this may be caused due to an unpinned dependency. Try replicating in a clean environment, I'll see if I can.

**Comment by stsewd** [comment]
I get the same result locally in a clean environment, although, at least there I can see some warnings.

```
WARNING: autodoc: failed to import module 'image' from module 'morphocut'; the following exception was raised:
No module named '_lzma'
looking for now-outdated files... none found
```

So, this doesn't look like a problem with rtd.

**Comment by moi90** [comment]
Thanks for having a look! I will try to investigate.

**Comment by moi90** [comment]
It seems to be a problem with Sphinx. Version ~=2.2 works, ~=3.3 not (although no error is shown). I will fix the version to ~=2.2 for now.
